api-report/container-definitions.api.md

```diff
@@ -85,7 +85,9 @@ export interface IAudienceOwner extends IAudience {
 // @public
 export interface IBatchMessage {
     // (undocumented)
-    contents: string;
+    compression?: string;
+    // (undocumented)
+    contents?: string;
```

Comment:
Is there a back-compat risk here about a runtime having to process an `IBatchMessage` with `undefined` `contents`? Aside from that, this API change should also be safe, this is more of an internal class anyway. @vladsud what do you think?

Reply:
IBatchMessage interface itself is internal (and ideally should not be exported out from package, or at least marked as @internal). So I do not think that's a concern.
That said, if ops on the wire are changed (and we start to use undefined where previous it was always a string), then it's a concern. But even there, if it's scoped only when compression is ON, then it's not a concern, as we know (and have to tell it very loudly to all consumers) that they can enable compression only after they have 99.99% saturation with the bits that understand compressison